github/model/model.go

```diff
@@ -51,19 +52,19 @@ func ToSvcGitTags(tags []*github.RepositoryTag) []svcmodel.GitTag {
 	return t
 }
 
-func ToSvcGithubRelease(release *github.RepositoryRelease) (svcmodel.GithubRelease, error) {
-	u, err := url.Parse(release.GetHTMLURL())
+func ToSvcGithubRelease(rls *github.RepositoryRelease, repo GithubRepo) (svcmodel.GithubRelease, error) {
+	u, err := url.Parse(rls.GetHTMLURL())
 	if err != nil {
 		return svcmodel.GithubRelease{}, err
 	}
 
 	return svcmodel.GithubRelease{
-		ID:          release.GetID(),
-		Name:        release.GetName(),
-		Body:        release.GetBody(),
-		HTMLURL:     *u,
-		TagName:     release.GetTagName(),
-		CreatedAt:   release.CreatedAt.Time,
-		PublishedAt: release.PublishedAt.Time,
+		ID:             rls.GetID(),
+		OwnerSlug:      repo.OwnerSlug,
+		RepositorySlug: repo.RepositorySlug,
+		GitTagName:     rls.GetTagName(),
+		HTMLURL:        *u,
+		CreatedAt:      rls.CreatedAt.Time,
+		UpdatedAt:      time.Now(),
```

Comment:
To keep things simple, I only store the necessary information in the GithubRelease model.

Since the `GithubRelease` name and body will be synced with `Release.ReleaseTitle `and `Release.ReleaseNotes`, it doesn't make sense to also save name and body in `GithubRelease`.

Reply:
Feels a bit too complicated. 😄 

Related to a huge comment below (or above), from a usage perspective, do we need any synchronization with GitHub Releases at all? Even the link to GitHub Release is unnecessary in both scenarios.